docs/integrations/catalog/prefect-shell.yaml

```
@@ -1,6 +1,6 @@
 collectionName: prefect-shell
 author: Prefect
 authorUrl: https://prefect.io
-documentation: https://prefecthq.github.io/prefect-shell/
+documentation: https://docs.prefect.io/integrations/prefect-shell/
```

Relative links should work here
```suggestion
documentation: prefect-shell
```